src/Markdig.Wpf/MarkdownExtensions.cs

```diff
@@ -25,7 +25,8 @@ public static MarkdownPipelineBuilder UseSupportedExtensions([NotNull] this Mark
                 .UseAutoLinks()
                 .UseGridTables()
                 .UsePipeTables()
-                .UseTaskLists();
+                .UseTaskLists()
+                .UseEmphasisExtras();
```

Comment:
I don't remember if the order matters. I think it does sometimes, so maybe  it would be better to follow the same order than https://github.com/lunet-io/markdig/blob/master/src/Markdig/MarkdownExtensions.cs#L59
Note that the order is already different so not a big deal at the moment.